step/serialization.md

```diff
@@ -1,109 +1,553 @@
-# Serialization holder
+# Serialization
 
 ## Overview
 
-This 5-minute optional lab will show you how **`jlink`** can greatly improve the size of a Java runtime image.
+Serialization has long been a sticky subject in Java. Brian Goetz describing it as a paradox, a key to Java's success allowing for transparent remoting, but also the implementation of serialization making a number of key mistakes. 
 
-...
+A short list of mistakes include;
```

Comment:
Btw I wouldn't use 'mistakes' but shortcomings or issues instead

Reply:
Yep, was going through and doing some edits and changed that to issues.